src/components/DocumentActions.tsx

```
@@ -81,44 +81,28 @@ const DocumentActions: React.FC<DocumentActionsProps> = ({
-				const imageEntries = Object.values(subject).filter(
-					(entry): entry is ImageEntry =>
-						entry &&
-						typeof entry === 'object' &&
-						typeof (entry as ImageEntry).mimetype === 'string' &&
-						(entry as ImageEntry)
-							.mimetype!.toLowerCase()
-							.startsWith('image/') &&
-						!!(entry as ImageEntry).content
-				);
+			if (subject && typeof subject === 'object') {
+				Object.values(subject).forEach((entry: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)